app/graphql/resolvers/integration_mappings/mappings_resolver.rb

```diff
@@ -2,7 +2,7 @@
 
 module Resolvers
   module IntegrationMappings
-    class NetsuiteMappingsResolver < Resolvers::BaseResolver
+    class MappingsResolver < Resolvers::BaseResolver
```

Comment:
Similar comment as the one for query - what about having just:

```
module Resolvers
  class IntegrationMappingsResolver
  ...
```

(and also `IntegrationMappingResolver`)